libs/java/client_common/src/main/java/com/yahoo/athenz/common/messaging/DomainChangeMessage.java

```diff
@@ -23,12 +23,14 @@
 public class DomainChangeMessage {
     
     // Represent the changed object type 
-    enum ObjectType {
+    public enum ObjectType {
         DOMAIN, 
-        ROLE, 
+        ROLE,
+        GROUP,
         POLICY, 
-        SERVICE, 
-        ENTITY
+        SERVICE,
+        ENTITY,
+        USER
```

Comment:
Not sure if this is the right object since we don't manage USERs. When we manage users, it's typically to delete members from roles/groups, etc. So I don't expect any notifications where a user object was modified. The correct events would be either a role or a group was modified in a given domain.